.github/workflows/analyze.yml

```diff
@@ -40,10 +45,8 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
     - uses: actions/download-artifact@v3
-    - name: Remove files in results bucket
-      run: aws s3 rm  s3://cellpack-results --recursive
     - name: Copy files to results bucket
-      run: aws s3 cp ./results s3://cellpack-results --recursive --acl public-read 
+      run: aws s3 cp ./results s3://cellpack-results/$GITHUB_REF_NAME/ --recursive --acl public-read 
```

Comment:
Looks good, and looks like it's working but I don't get where GITHUB_REF_NAME is being defined

Reply:
From reading i think $GITHUB_REF_NAME is analagous to "{{ github.ref_name }}". Using it this way to make it more easily readable and able to be used with strings well